README.md

```diff
@@ -1,17 +1,20 @@
-# EXPERIMENTAL/UNDER DEVELOPMENT/IN NEED OF LOVE
+# Digital Marketplace Runner ('DMRunner')
 
-## Digital Marketplace Runner
-A utility to run your API/frontend repositories together (and backing services if required), while allowing you to
-interact with the running processes and their logs. This script is primarily compatible with macOS and Ubuntu 16.04;
-running against other OSs may require some care and consideration. At its simplest, as an existing developer, you should
-be able to clone this repo, run `make setup` and follow the instructions, then just run `make` to bring up all your
-checked out apps.
+** EXPERIMENTAL / UNDER DEVELOPMENT / IN NEED OF LOVE **
+
+A local development environment for running Digital Marketplace applications and backing services.
+
+DMRunner also provides an interactive shell to manage processes and their logs.
 
 ## Requirements
-You must have the following tools available in order to successfully use the DM Runner:
+
+DMRunner is compatible with macOS and Ubuntu 16.04.
+
+Running against other OSs may require some care and consideration.
 
 * Python 3, including headers if appropriate (consider using [pyenv]),
   installed with `pip` and `virtualenv` packages.
+  * macOS/Homebrew users can install with `brew install pyenv && pyenv shell 3.6.8`
```

Comment:
am not sure this will _just work_ on a clean machine as they would still need to add something to their `~/.bash_profile` or  equivalent as per: https://github.com/pyenv/pyenv#installation

Reply:
Added an extra commit with an example to put in `.bash_profile`.